buildchain/buildchain/docker_command.py

```diff
@@ -63,11 +63,18 @@ def build_error_handler(build_error: BuildError) -> str:
         if "stream" in item:
             line = item["stream"]
         elif "status" in item:
-            line = "{}: {}/{}".format(
-                item["status"],
-                item["progressDetail"]["current"],
-                item["progressDetail"]["total"],
-            )
+            try:
+                line = "{0}: ".format(item["id"])
+            except KeyError:
+                line = ""
+            line += item["status"]
+            try:
+                line += ": {0}/{1}\x1b[1K\r".format(
+                    item["progressDetail"]["current"],
+                    item["progressDetail"]["total"],
+                )
+            except KeyError:
+                line += "\n"
```

Comment:
Not sure to understand why we only add a break line if we do not have progressDetail

Reply:
If we display progressDetail, we want it to be behave like it's done on CLI output, re-writring previous progress line with the new one, otherwise we want to add a newline for the sake of readability (to avoid having everything on a single line).

Reply:
We could also not display this progressDetail at all, since normally it's overwritten by the next line (e.g. 'Verifying Checksum'), it's just in case the download/extract would be stopped in the middle.

Reply:
Oh ok, got it, thank you for the explanation :ok_hand: